utils/auto_annotation/README.md

```diff
@@ -50,3 +50,16 @@ $ python cvat/utils/auto_annotation/run_modely.py --py /path/to/python/interp.py
                                                   --show-images
                                                   --show-image-delay 2000
 ```
+
+Visualization isn't always enough. The CVAT has a serialization step that can throw confusing errors on model upload.
```

Comment:
@benhoff , why not change the error message on the server side? Make it more friendly and avoid the extra code in the script?

Reply:
See my below response, but I've personally find it difficult to iteratively develop issues with these scripts with the web interface.

Reply:
@nmanovic I also tweaked the verbiage to more accurately reflect why you would need/want to do this.